### PR TITLE
Story/wics 10

### PR DESCRIPTION
**Pull request by MaulikLim**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 

[WICS-10] - If there is a text for a concept, it needs to be displayed on the public concept page.
  
## Anything else the reviewer needs to know?

... describe here ...

[WICS-10]: https://diging.atlassian.net/browse/WICS-10?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
The title should be more clearly a title (maybe a `h3`? whatever looks right)
image
And let's add a little bit more space between the network and the relations and add a heading "Relationships" above the relations table.
image
Also the author of the text is missing. Below the title it should say "By [Author]".

**Comment by MaulikLim** [comment]
This merge request is also inclusive of [WICS - 12](https://diging.atlassian.net/browse/WICS-12?atlOrigin=eyJpIjoiZjViOWE4YzYwODI1NGQ0MDhmNjY3MTg2NTBjNTVhMjMiLCJwIjoiaiJ9)

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: wic/src/main/webapp/WEB-INF/views/person.jsp [hunk 1]
```diff
@@ -58,31 +68,19 @@
...
+        <h3><strong>${text.title}</strong></h3>
+        <figcaption>By ${text.author}</figcaption>
+        <p>${text.htmlRenderedText()}</p>
+    </li>
+    </c:forEach>
+    </ul>
 </c:if>
-<c:if test="${not isPerson}">
+<c:if test="${texts == null or texts.isEmpty()}">
+    <h5 id="relationships">Relationships</h5>
```
Comment (jdamerow) [inline wic/src/main/webapp/WEB-INF/views/person.jsp:83]:
```suggestion
    <h4 id="relationships">Relationships</h4>
```

File: wic/src/main/webapp/WEB-INF/views/person.jsp [hunk 2]
```diff
@@ -93,7 +91,8 @@ Quisque id mi. Ut tincidunt tincidunt erat. Etiam feugiat lorem non metus. Vesti
     <div id="spinner" class="text-center"><div class="fa fa-spinner fa-spin"></div> Loading graph...</div>
     <div id="network" style="min-width: 200px; min-height: 200px; "></div>
     
-    <c:if test="${isPerson}">
+    <c:if test="${texts != null and not texts.isEmpty()}">
+    <h5 id="relationships">Relationships</h5>
```
Comment (jdamerow) [inline wic/src/main/webapp/WEB-INF/views/person.jsp:95]:
```suggestion
    <h4 id="relationships">Relationships</h4>
```

Comment (MaulikLim) [reply wic/src/main/webapp/WEB-INF/views/person.jsp]: Made the change

File: wic/src/main/webapp/WEB-INF/views/person.jsp (hunk 1 above)
Comment (MaulikLim) [reply wic/src/main/webapp/WEB-INF/views/person.jsp]: done

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Pull request merged**